omegaconf/omegaconf.py

```diff
@@ -411,7 +410,7 @@ def resolver_wrapper(
     def register_new_resolver(
         name: str,
         resolver: Resolver,
-        use_cache: Optional[bool] = True,
+        use_cache: Optional[bool] = False,
     ) -> None:
```

Comment:
make use_cache keyword only:
```suggestion
        *,
        use_cache: Optional[bool] = False,
    ) -> None:
```

I suspect we will want to add support for a more flexible caching policy later and this will make it easier.

Reply:
Good idea => done in 3a76edc